package.json

```diff
@@ -71,7 +72,14 @@
     "glob": "7.2.3",
     "husky": "7.0.4",
     "jest": "27.5.1",
+    "patch-package": "8.0.0",
     "resize-observer-polyfill": "^1.5.1",
-    "rosie": "2.1.1"
+    "rosie": "2.1.1",
+    "sharp": "0.32.6"
+  },
+  "overrides": {
+    "@edx/frontend-build": {
+      "sharp": "$sharp"
+    }
```

Comment:
[inform] It looks like the upstream revert [PR](https://github.com/openedx/frontend-build/pull/512) to get `sharp` back to `0.32.6` was merged upstream in latest version of `@openedx/frontend-build`. Likely still need to keep this override here, unless it's possible to migrate to the `@openedx/frontend-build` scope.

Reply:
I saw! Glad it already got addressed upstream. I think for this PR we'll want to keep the override, and depending on how urgently we want to handle the `@edx` --> `@openedx` migration we'd be able to remove this override and the patches. Arguably, it'd be a good reason to do a follow-up ticket to migrate Learner Record and any other MFEs we want to experiment JS prod builds with.